administrator-manual/en/nextcloud.rst

```diff
@@ -44,6 +50,18 @@ in section `Internal Username` of `Official NextCloud documentation <https://doc
                 a dedicated user account in AD is required by the module to be fully
                 operational! See :ref:`join-existing-ad-section`.
 
+Custom Virtual Host
+===================
+
+It is used to customize web url to access nextcloud instance. To customize: ::
+
+    config setprop nextcloud VirtualHost mynextcloud.domain.com
+    config setprop nextcloud TrustedDomains mynextcloud.domain.com
+    signal-event nethserver-nextcloud-update
+
+If you use :ref:`let's encrypt <server_certificate-section>` remember to add the domain name to its list.
```

Comment:
If you use :ref:`let's encrypt <server_certificate-section>` remember to add the domain name to the proper list